### ISSUE TITLE
self hosting doesn't work

### ISSUE THREAD

**Issue by Twilight** [dimok789/loadiine_gx2#134]
5.5.1, 0dffac5, browser stuck, have to power off by holding the power button

homebrew launcher in a different path works
http://xxx.xxx/wiiu/homebrew_launcher/
http://xxx.xxx/wiiu/loadiine_gx2/
hosted in a Debian 8 box, lighttpd, php5

and loadiine.ovh loadine GX2 0.3 SD(...) works too.

I can still launch loadiine in homebrew launcher so not that big of a deal, but still, why?

**Comment by CreeperMario** [comment]
From what I understand, HBL will launch from Loadiine.ovh and from your Debian box.
Loadiine GX2 will launch from Loadiine.ovh, but not from your Debian box.
Is that correct?

**Comment by Twilight** [comment]
Yes, exactly

**Comment by CreeperMario** [comment]
If HBL can launch through self-hosting, then that's all you need.
Loadiine GX2 can be launched through HBL.

**Comment by Twilight** [comment]
yeah like I said in the original post, not that big of a deal, but all of a sudden HBL stopped working :(

loadiine.ovh still works, both HBL and loadiine.

[drizzt's static payload](https://github.com/drizzt/wiiu-payloads/) doesn't work.

so I dug a little trying to figure out why, turns out, loadiine.ovh uses somewhat different payloads, while drizzt's static payloads are the same to payloads generated from php, byte exact.

I downloaded loadiine.ovh's payloads and hosted them on the same Debian box with lighttpd, it works, both HBL and loadiine GX2.
